Backend/WebAPI/Areas/HelpPage/Views/Web.config

```diff
@@ -9,7 +9,7 @@
   </configSections>
 
   <system.web.webPages.razor>
-    <host factoryType="System.Web.Mvc.MvcWebRazorHostFactory, System.Web.Mvc, Version=5.2.4.0, Culture=neutral, PublicKeyToken=31BF3856AD364E35" />
```

Comment:
This WebAPI project will be replaced by a clean Web Api project that gets rid of unnecessary MVC packages. Refer to bug branch on the SSO repo